docs/team.md

```diff
@@ -1,9 +1,139 @@
+</figcaption>
+</figure>
+
+<figure class="bio" markdown>
+![Will Scott](mugshots/will.jpg){ width="300" }
+<figcaption markdown>### Will Scott
+#### Glacial Isostatic Adjustment
+
+---
+
+[Will](https://researchers.anu.edu.au/researchers/scott-w) is a computational geoscientist at the ANU's Research School of Earth Sciences, funded as a post-doctoral researcher by the Australian Centre for Excellence for Antarctic Science (ACEAS). Will's focus is on developing next generation (forward and inverse) finite element models to better understand polar contributions to sea level change. Before joining ANU he developed a new model for ocean flow under ice shelf cavities in Antarctica using the Firedrake finite element framework. At ANU he is working as part of a cross-disciplinary team to develop a 'state-of-the-art' Glacial Isostatic Adjustment (GIA) model to reduce uncertainty in sea level projections.
+</figcaption>
+</figure>
+</div>
+
+## Scientific Advisors
+
+<div class="bio-grid" markdown>
+<figure class="bio" markdown>
+![Mark Hoggard](mugshots/mark.jpg){ width="300" }
+<figcaption markdown>### Mark Hoggard
+#### G-ADOPT Focus: Glacial Isostatic Adjustment
+
+---
+
+[Mark](https://mjhoggard.com/) is a Discovery Early Career Research Fellow ([DECRA](https://www.arc.gov.au/funding-research/funding-schemes/discovery-program/discovery-early-career-researcher-award-decra)) at the ANU's Research School of Earth Sciences. His research program is generally problem-orientated and therefore spans a wide range of topics, crossing the traditional discipline boundaries of Earth sciences. It brings together data and models from across seismology, rock physics, geochemistry, sedimentology, geochronology, geomorphology and paleoclimate. It combines field observations, numerical modelling, high-performance computing, uncertainty propagation and statistics. Mark generally tries to take an inverse approach to solving problems, whereby observational datasets are used to place constraints on unknown physical properties and parameters of the Earth. 
+</figcaption>
+</figure>
+
+<figure class="bio" markdown>
+![Matt King](mugshots/matt_king.jpg){ width="300" }
+<figcaption markdown>### Matt King
+#### G-ADOPT Focus: Glacial Isostatic Adjustment
+
+---
+
+[Matt](https://discover.utas.edu.au/Matt.King) is Professor of polar geodesy at the University of Tasmania. ACEAS Director. Since 2021, he has been Director of the [Australian Centre for Excellence in Antarctic Science](https://antarctic.org.au/). His research focusses on solid earth geophysics and geodesy, with a particular focus on Antarctica. Matt focuses on the use of geodetic tools to solve problems related to Earth geophysics, notably Earth deformation due to glacier melt, earthquakes, and tides; sea-level change; and polar ice mass balance. He also seeks to advance the accuracy and precision of those geodetic tools (e.g., Global Navigation Satellite Systems (GNSS/GPS) and satellite gravimetry (GRACE)).
```

Comment:
```suggestion
[Matt](https://discover.utas.edu.au/Matt.King) is Professor of polar geodesy at the University of Tasmania. Since 2021, he has been Director of the [Australian Centre for Excellence in Antarctic Science](https://antarctic.org.au/). His research focusses on solid earth geophysics and geodesy, with a particular focus on Antarctica. Matt focuses on the use of geodetic tools to solve problems related to Earth geophysics, notably Earth deformation due to glacier melt, earthquakes, and tides; sea-level change; and polar ice mass balance. He also seeks to advance the accuracy and precision of those geodetic tools (e.g., Global Navigation Satellite Systems (GNSS/GPS) and satellite gravimetry (GRACE)).
```

Reply:
Good catch!